src/components/LoginPage/Slide3.js

```diff
@@ -0,0 +1,164 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import arrow from 'components/LoginPage/images/arrow.svg'
+import imageNoteContent from 'components/LoginPage/images/slide-3-note-content.svg';
+import imageMindmapExplanation from 'components/LoginPage/images/slide-3-mindmap-explanation.svg';
+import { Img, Caption } from 'components/LoginPage/slideComponents';
+
+const Slide = styled.section`
+  padding-bottom: 0%;
+  text-align: center;
+  .note-example {
+    width: 61%;
+  }
+   .edit-example {
+    position: relative;
+    margin-top: -9%;
+  }
+  .container-for-hint {
+    margin-top: -65%;
+    margin-bottom: 59%;
+    position: relative;
+  }
+  .hint-attach-files {
+    width: 22%;
+    padding-top: 3%;
+    padding-bottom: 3%;
+  }
+  .subheader {
+    margin-top: -53%;
+    margin-bottom: 45%;
+    position: relative;
+  }
+`;
+const H2 = styled.h2`
+  padding-bottom: 7%;
+  font-size: calc(16px + (30 - 16)*(100vw - 375px)/(1200 - 375));
+  text-align: center;
+  color: #ffffff;
+`;
+const H3 = styled.h3`
+  padding-bottom: 7%;
+  text-align: center;
+  color: #ffffff;
+`;
+const Figure = styled.figure`
+  position: relative;
+  width: 74%;
+  margin: auto;
+  height: auto;
+`;
+const ArrowUp = styled.img`
+  width: 2%;
+  height: auto;
+`;
+const ArrowDown = styled.img`
+  width: 2%;
+  height: auto;
+  transform: rotate(180deg);
+`;
+const ArrowLeft = styled.img`
+  width: 1.5%;
+  height: auto;
+  transform: rotate(270deg);
+  position: absolute;
+  left: 36%;
+  top: 38%;
+`;
+const ArrowRight = styled.img`
+  width: 1.5%;
+  height: auto;
+  transform: rotate(90deg);
+  position: absolute;
+  right: 36%;
+  top: 38%;
+`;
+const Hint = styled.div`
+  position: relative;
+  width: 34%;
+  background: #0E1736;
+  border-radius: 5px;
+  box-shadow: inset 0px 4px 2px rgba(0, 0, 0, 0.25);
+  padding-left: 2.7%;
+  padding-right: 2.7%;
+  padding-top: 4%;
+  padding-bottom: 4%;
+  color: #E8E9F8;
+  text-transform: uppercase;
+  font-weight: 700;
+  margin-top: 1vw;
+  margin-bottom: 1vw;
+  margin-left: auto;
+  margin-right: auto;
+`;
+
+const CaptionTextSlide3 = ({x = 0, y = 15, fill = "#FFFFFF", fontWeight = 700, text}) => {
+  return <text x={x} y={y} fill={fill} font-weight={fontWeight}>{text}</text>;
+};
+
+const CaptionBoxSlide3 = ({viewBoxWidth = 100, viewBoxHeight = 21, ...rest}) => {
+  return <Caption viewBox={`0 0 ${viewBoxWidth} ${viewBoxHeight}`} {...rest} />;
+};
+
+
+
+export default class Slide3 extends Component {
+    render() {
+        return (
+            <Slide>
+            <H2><b>Edit Notes Content</b>
```

Comment:
wrong indentation: should be 2 or 4 spaces more. Right now all direct children of the `<Slide>` tag have the same indentation as the `<Slide>` tag itself.